doc/source/getting_started/using_xinference.rst

```diff
@@ -44,7 +44,8 @@ To start a local instance of Xinference, run the following command:
 
   .. code-block:: bash
 
-    XINFERENCE_HOME=/tmp/xinference xinference-local --host 0.0.0.0 --port 9997
+    export XINFERENCE_HOME=/tmp/xinference 
+    xinference-local --host 0.0.0.0 --port 9997
```

Comment:
You should `pip install -e .` to install `xinference-local` command. So here should not be changed. It is right.

Reply:
![image](https://github.com/xorbitsai/inference/assets/57928993/749bc4bc-1012-4113-bb8d-fd57cf8a73dd)